README.md

```diff
@@ -226,6 +226,8 @@ The second most obvious is to generate docs via a Rake task. You can do this by
 adding the following to your `Rakefile`:
 
 ```ruby
+require 'yard/rake/yardoc_task'
```

Comment:
Switching this to `yard` fixes the issue around lazy loading below.

```suggestion
require 'yard'
```

Reply:
Ok.